docs/source/commands.rst

```diff
@@ -208,3 +209,40 @@ pair is equal or greater to the count, no new invalidations will be sent.
    * - ``--count INTEGER``
      - Number of invalidations to create, per (PROOF_SOURCE, PROOF_TARGET) pair.
        Has to be greater or equal to 1. Default: 1.
+
+
+.. _command-check-verify-l1-invalidations:
+
+``beamer check verify-l1-invalidations``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``beamer check verify-l1-invalidations [OPTIONS] FILE``
+
+The command verifies L1 invalidations contained in ``FILE``. In case of
+multiple invalidations for the same pair ``(proof_source, proof_target)``, if
+an invalidation is verified successfully, remaining invalidations for the same
+chain pair will be skipped as they then not considered necessary. If a
```

Comment:
missing `are` ? `as they then not considered necessary`

Reply:
> missing `are` ? `as they then not considered necessary`

Nice catch! :eyes: 
Fixed, plus I rewored the last sentence to be more clear:
```diff
diff --git a/docs/source/commands.rst b/docs/source/commands.rst
index cdace38c..a5f2955c 100644
--- a/docs/source/commands.rst
+++ b/docs/source/commands.rst
@@ -221,10 +221,11 @@ pair is equal or greater to the count, no new invalidations will be sent.
 The command verifies L1 invalidations contained in ``FILE``. In case of
 multiple invalidations for the same pair ``(proof_source, proof_target)``, if
 an invalidation is verified successfully, remaining invalidations for the same
-chain pair will be skipped as they then not considered necessary. If a
+chain pair will be skipped as they are then not considered necessary. If a
 verification fails for any invalidation, for whatever reason, an error will be
 emitted and the process will continue with the next invalidation. The command
-will exit with a success code only if there were no errors.
+will exit with a success code only if all chain pairs had at least one
+successful invalidation.
```